src/apps/capitals.jsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import capitalsData from "./capitalsData.json";
+
+const Territory = styled.div`
+  text-transform: capitalize;
+`;
+
+const Capital = styled.div`
+  text-transform: capitalize;
+  font-weight: bold;
+`;
+
+//=========================================
+
+function Renderer({ data }) {
+  return (
+    <div>
+      Capital of <Territory>{data.territory}</Territory> is{" "}
+      <Capital>{data.capital}</Capital>
+    </div>
+  );
+}
+
+function territoryToCapital(territory) {
+  return capitalsData.find(
+    (elem) => elem.CountryName.toLowerCase() === territory.toLowerCase()
+  )?.CapitalName;
+}
+
+const queryToTerritory = async ({ query }) => {
+  const normalizedQuery = query.toLowerCase();
+  // Todo @Harsh: There is probably a smarter way to do this
+  // Todo @Harsh: Replace this with a standard utility function
+  const triggerQueries = [
+    "what is the capital of",
+    "what is capital of",
+    "capital of",
+    "capital",
+  ];
+  for (const triggerQuery of triggerQueries) {
+    if (normalizedQuery.startsWith(triggerQuery)) {
```

Comment:
Missed opportunity for serving "Sweden capital". 😅 

<img width="786" alt="Screenshot 2021-09-16 at 1 29 21 PM" src="https://user-images.githubusercontent.com/8065913/133604467-162592da-5994-4a59-a636-538c60340695.png">

Reply:
aah yes, it should "includes"

Reply:
Fixed this